### PR TITLE
Add endpoint to get MTUs by update UUID.

### PR DESCRIPTION
**Pull request by houcros**

### CODE REVIEW

**Comment by jochenschneider** [comment]
Please add the ticket to the branch name (e.g., "feat/OTA-123/add-get-mtu-endpoint").

File: src/test/scala/com/advancedtelematic/director/http/AdminResourceSpec.scala [hunk 1]
```diff
@@ -196,6 +197,14 @@ trait AdminResourceSpec extends DirectorSpec with KeyGenerators with DeviceRegis
 
     fetchRootOk(1).signed shouldBe fetchRootOk.signed
   }
+
+  testWithNamespace("multi_target_update/updateId gives all the MTUs in the namespace") { implicit ns =>
+    val newMTURequest = GenMultiTargetUpdateRequest.generate
+    val updateId = createMultiTargetUpdateOK(newMTURequest)
+
+    val foundUpdates = findByUpdate(updateId)
+    foundUpdates.map(_.id).distinct.loneElement should be(updateId)
```
Comment (jochenschneider) [inline src/test/scala/com/advancedtelematic/director/http/AdminResourceSpec.scala]: Didn't know about `loneElement`, nice!

Comment (houcros) [reply src/test/scala/com/advancedtelematic/director/http/AdminResourceSpec.scala]: All that time I spent reading docs finally payed off! haha

File: src/main/scala/com/advancedtelematic/director/data/Codecs.scala [hunk 2]
```diff
@@ -41,6 +41,9 @@ object Codecs {
   implicit val decoderCustomImage: Decoder[CustomImage] = deriveDecoder
   implicit val encoderCustomImage: Encoder[CustomImage] = deriveEncoder
 
+  implicit val multiTargetUpdateRowDecoder: Decoder[MultiTargetUpdateRow] = deriveDecoder
```
Comment (simao) [inline src/main/scala/com/advancedtelematic/director/data/Codecs.scala]: MultiTargetUpdateRow is a db data type and should not ever leak to the client, hence the Row at the end of the type.

I think we should create a new datatype for this if one does not exist yet, take a look at `MultiTargetUpdateRequest`, that is what the client uses to give us MTUs, we should probably deliver something similar. IIRC we already use `LaunchedMultiTargetUpdate` to return launched MTUs, maybe we can reuse that

Comment (houcros) [reply src/main/scala/com/advancedtelematic/director/data/Codecs.scala]: Changed return type to `MultiTargetUpdateRequest ` so we return the same format that we get.

**Comment by houcros** [comment]
Will close this and open new PR with proper branch name.

**Pull request closed without merging**